.circleci/config.yml

```diff
@@ -13,101 +13,120 @@ jobs:
           keys:
           - frontend-deps-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
       - run:
-          name: Install dependencies
-          command: |
-            make bootstrap
-      - run:
-          name: Build styles
-          command: |
-            make build-css
+          name: Install dependencies & build styles
+          command: make bootstrap
       - save_cache:
           paths:
             - ~/pixel/static
             - ~/pixel/node_modules
           key: frontend-deps-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
 
-  build_backend:
-    working_directory: ~/pixel
-    docker:
-      - image: circleci/python:3.6.1
-        environment:
-          PIPENV_VENV_IN_PROJECT: 1
-          CI_BUILD_BACKEND: 1
+  backend-build:
+    machine: true
     steps:
       - checkout
       - restore_cache:
           keys:
-          - backend-deps-v3-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
-      # In user mode, Pipenv is installed to $HOME/.local/bin
+          - frontend-deps-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
       - run:
-          name: Add Pipenv to the PATH
-          command: |
-            echo "export PATH=~/.local/bin:$PATH" >> $BASH_ENV
+          name: Login to DockerHub
+          command: docker login -u $DOCKER_USER -p $DOCKER_PASS
       - run:
-          name: Install dependencies
-          command: |
-            pip3 install --user pipenv
-            make bootstrap
+          name: Build container
+          command: docker build -t candihub/pixel:$CIRCLE_BRANCH .
+      - run:
+          name: Publish container
+          command: docker push candihub/pixel:$CIRCLE_BRANCH
```

Comment:
I am still a bit worried because it seems there is no way to delete images on DockerHub...

Reply:
I haven't tested yet, but I mentioned a workaround in #43. WDYT?

Reply:
it is good but it seems it has not been implemented in dockerhub, there is no proper way to remove images apparently..